modules/fedimint-ln-client/src/receive.rs

```diff
@@ -38,11 +39,12 @@ const RETRY_DELAY: Duration = Duration::from_secs(1);
 /// ```
 #[derive(Debug, Clone, Eq, PartialEq, Hash, Decodable, Encodable)]
 pub enum LightningReceiveStates {
-    SubmittedOffer(LightningReceiveSubmittedOffer),
+    SubmittedOfferV0(LightningReceiveSubmittedOfferV0),
```

Comment:
I think with the new migration code we can now just replace that v0 variant with the updated one?

Reply:
Ideally, yes, but unfortunately it makes testing harder. I made this as a follow up, which would allow us to delete states https://github.com/fedimint/fedimint/issues/4536

Reply:
Ok, then we should at least open an issue/leave a FIXME that this has to be cleaned up as soon as we can test it.